internal/docker/builder.go

```diff
@@ -457,7 +509,32 @@ func deployImage(
 	if err != nil {
 		return nil, err
 	}
+
 	containerID := body.ID
+
+	// Create the application service files
+	for asID, registration := range asIDToRegistrationMap {
+		var buf bytes.Buffer
+		tw := tar.NewWriter(&buf)
+		err = tw.WriteHeader(&tar.Header{
+			Name: fmt.Sprintf("/appservices/%s.yaml", asID),
+			Mode: 0777,
+			Size: int64(len(registration)),
+		})
+		if err != nil {
+			return nil, fmt.Errorf("Failed to copy regstration to container: %v", err)
+		}
+		tw.Write([]byte(registration))
+		tw.Close()
```

Comment:
This is from https://stackoverflow.com/a/52131297/796832

Reply:
Please chuck that link as a comment with a short prose to explain what this code is doing. It's unclear why we're doing this otherwise, but I do like that we can do this!